website/docs/contract-test-collection.md

```diff
@@ -146,24 +252,108 @@ storage (pair
           (nat %total_supply));
 ```
 
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+const mapAccount1 = new MichelsonMap();
+mapAccount1.set("tz1h3rQ8wBxFd8L9B3d7Jhaawu6Z568XU3xY", '25');
+mapAccount1.set("tz1NhNv9g7rtcjyNsH8Zqu79giY5aTqDDrzB", '25');
+
+const mapAccount2 = new MichelsonMap();
+mapAccount2.set("tz3WXYtyDUNL91qfiCJtVUX746QpNv5i5ve5", '25');
+mapAccount2.set("tz1KqTpEZ7Yob7QbPE4Hy4Wo8fHG8LhKxZSx", '25');
+
+const bigMapLedger = new MichelsonMap();
+bigMapLedger.set("tz1PgQt52JMirBUhhkq1eanX8hVd1Fsg71Lr"), {
+  balance: '50',
+  allowances: mapAccount1,
+});
+bigMapLedger.set(user_addresses.get("tz1Nu949TjA4zzJ1iobz76fHPZbWUraRVrCE"), {
+  balance: '50',
+  allowances: mapAccount2,
+});
+
+const op = await tezos.contract.originate({
```

Comment:
I don't think the storage is correct, it should have a `token_metadata` MichelsonMap

Reply:
I found the correct way to show this storage:
```
    const bigMapLedger = new MichelsonMap();
    bigMapLedger.set('tz1c1X8vD4pKV9TgV1cyosR7qdnkc8FTEyM1', {
      allowances: ['tz1h3rQ8wBxFd8L9B3d7Jhaawu6Z568XU3xY'],
      balance: '50'
    });
    bigMapLedger.set('tz1XTyqBn4xi9tkRDutpRyQwHxfF8ar4i4Wq', {
      allowances: ['tz1Nu949TjA4zzJ1iobz76fHPZbWUraRVrCE'],
      balance: '50',
    });

    const tokenMetadataBigMap = new MichelsonMap();
    tokenMetadataBigMap.set('0', {
      token_id: '0',
      symbol: 'hello',
      name: 'test',
      decimals: '0',
      extras: new MichelsonMap()
    });
    tokenMetadataBigMap.set('1', {
      token_id: '1',
      symbol: 'world',
      name: 'test2',
      decimals: '0',
      extras: new MichelsonMap()
    });

    const op = await tezos.contract.originate({
      balance: "1",
      code: fa2Contract,
      storage: {
        ledger: bigMapLedger,
        token_metadata: tokenMetadataBigMap,
        total_supply: '100'
      },
    })
```